chore(doc): add nacos configuration manual

whalecold opened the pull request:
#### What type of PR is this?
<!--
Add one of the following kinds:

build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
ci: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
docs: Documentation only changes
feat: A new feature
optimize: A new optimization
fix: A bug fix
perf: A code change that improves performance
refactor: A code change that neither fixes a bug nor adds a feature
style: Changes that do not affect the meaning of the code (white space, formatting, missing semi-colons, etc)
test: Adding missing tests or correcting existing tests
chore: Changes to the build process or auxiliary tools and libraries such as documentation generation
-->

#### Check the PR title.
<!--
The description of the title will be attached in Release Notes, 
so please describe it from user-oriented, what this PR does / why we need it.
Please check your PR title with the below requirements:
-->
- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.
- [x] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)


#### (Optional) Translate the PR title into Chinese.


#### (Optional) More detailed description for this PR(en: English/zh: Chinese).
<!--
Provide more detailed info for review(e.g., it's recommended to provide perf data if this is a perf type PR).
-->
en:
zh(optional): 


#### (Optional) Which issue(s) this PR fixes:
<!--
Automatically closes linked issue when PR is merged.
Eg: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->

#### (optional) The PR that updates user documentation:
<!--
If the current PR requires user awareness at the usage level, please submit a PR to update user docs. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)
-->

felix021 commented on README_CN.md:
````diff
@@ -115,16 +115,131 @@ func main() {
+```
+configDataID: .echo.limit
+{
+  "connection_limit": 11, // 最大100并发
````
11？100？

whalecold replied:
Done.

felix021 commented on README.md:
```diff
@@ -32,8 +32,7 @@ type EchoImpl struct{}
 	klog.Info("echo called")
-	return &api.Response{Message: req.Message}, nil
-}
+	return &api.Response{Message: req.Message}, nil }
```
这个换行？

felix021 commented:
1. [p2] DefaultClient() 目前完全是从 ENV 中取，最好提供一种方式可以通过代码设置
2. [p1] 指定 server Suite 的方式：server.WithSuite(nacosserver.NewSuite(serviceName, nacosClient)); client用 client.WithSuite
3. [p0] Kitex 依赖更新到 v0.7.1 (已发布)
4. [p2] limit 的 dataId(.echo.limit) 缺失了 clientName（空串），看起来有点别扭，考虑直接去掉开头的点?
5. [p1] CustomFunction 的用法没有说明（例子里给了个空函数，比较奇怪），建议例子里不体现，但另开一小节介绍一下，或者在 example 里给个具体的例子
6. [p1] 环境变量名称用小写不合适，一般是XX_YY_ZZ，且应添加前缀避免重名，例如 KITEX_CONFIG_NACOS_SERVER_ADDR
7. [p1] 在配置示例中，代码块内仅放 json （方便用户拷贝直接使用），注释（无法正常unmarshal）和其他说明放在代码块前/后
8. [p1] 具体例子中，clientName 和 serverName 用 "ClientName" 和 "ServerName"，辨识度更高，用户不容易误解（现在文档中client是test，但配置示例里是 requester，用户较难理解）
9. [p1] 请求方法名称似乎不需要转小写？我的例子里 ri.To().Method() 是和 idl 里一致的
10. [p2] 对于nacos中没配的key，能否不打 error log ? 没配用默认值，打error log容易引起用户误解，以为是出错了.(看起来是nacos-sdk打的?)
    > [Error] api</v1/cs/configs>,method:<GET>, params:<{"dataId":"test.echo.circuit_break","group":"DEFAULT_GROUP","type":"json"}>, call domain error:<[404] config data not exist> , result:<config data not exist>
    2023/09/12 00:25:31.608710 logger.go:54: [Error] get config from server error:[404] config data not exist
11. [p0] 例子里的backup request（type=1）有错(从之前的proposal拷贝过来的?)：(1)policy key写错了，应是 backup_policy (2) backup retry的最大重试次数为2 (3) 没有设置 retry_delay_ms。一个正确的例子
```json
{
    "EchoSleep": {
        "enable": true,
        "type": 1,
        "backup_policy": {
            "retry_delay_ms": 100,
            "stop_policy": {
                "max_retry_times": 2,
                "max_duration_ms": 2011,
                "cb_policy": {
                    "error_rate": 0.2
                }
            }
        }
    }
}
```
12. [p2] retry 在 onChangeCallback 里没有和之前的配置比较（相同的可跳过update）; cb & timeout 也一样

whalecold commented:
> 1. [p2] DefaultClient() 目前完全是从 ENV 中取，最好提供一种方式可以通过代码设置
> 2. [p1] 指定 server Suite 的方式：server.WithSuite(nacosserver.NewSuite(serviceName, nacosClient)); client用 client.WithSuite
> 3. [p0] Kitex 依赖更新到 v0.7.1 (已发布)
> 4. [p2] limit 的 dataId(.echo.limit) 缺失了 clientName（空串），看起来有点别扭，考虑直接去掉开头的点?
> 5. [p1] CustomFunction 的用法没有说明（例子里给了个空函数，比较奇怪），建议例子里不体现，但另开一小节介绍一下，或者在 example 里给个具体的例子
> 6. [p1] 环境变量名称用小写不合适，一般是XX_YY_ZZ，且应添加前缀避免重名，例如 KITEX_CONFIG_NACOS_SERVER_ADDR
> 7. [p1] 在配置示例中，代码块内仅放 json （方便用户拷贝直接使用），注释（无法正常unmarshal）和其他说明放在代码块前/后
> 8. [p1] 具体例子中，clientName 和 serverName 用 "ClientName" 和 "ServerName"，辨识度更高，用户不容易误解（现在文档中client是test，但配置示例里是 requester，用户较难理解）
> 9. [p1] 请求方法名称似乎不需要转小写？我的例子里 ri.To().Method() 是和 idl 里一致的
> 10. [p2] 对于nacos中没配的key，能否不打 error log ? 没配用默认值，打error log容易引起用户误解，以为是出错了.(看起来是nacos-sdk打的?)
>     > [Error] api</v1/cs/configs>,method:, params:<{"dataId":"test.echo.circuit_break","group":"DEFAULT_GROUP","type":"json"}>, call domain error:<[404] config data not exist> , result:
>     > 2023/09/12 00:25:31.608710 logger.go:54: [Error] get config from server error:[404] config data not exist
> 11. [p0] 例子里的backup request（type=1）有错(从之前的proposal拷贝过来的?)：(1)policy key写错了，应是 backup_policy (2) backup retry的最大重试次数为2 (3) 没有设置 retry_delay_ms。一个正确的例子
> 
> ```json
> {
>     "EchoSleep": {
>         "enable": true,
>         "type": 1,
>         "backup_policy": {
>             "retry_delay_ms": 100,
>             "stop_policy": {
>                 "max_retry_times": 2,
>                 "max_duration_ms": 300,
>                 "cb_policy": {
>                     "error_rate": 0.2
>                 }
>             }
>         }
>     }
> }
> ```
> 
> 12. [p2] retry 在 onChangeCallback 里没有和之前的配置比较（相同的可跳过update）; cb & timeout 也一样

1. Fixed
2. Fixed
3. Done
4. Done
5. Done
6. Done
7. Done
8. Done
9. TODO
10. sdk 自己打的～
11. Done
12. TODO

felix021 commented on nacos/nacos.go:
```diff
@@ -69,6 +121,56 @@ func (c *client) SetParser(parser ConfigParser) {
+		return param, err
+	}
+
+	// TODO trim the specified prefix string
```
This way to trim dots is a bit tricky, and may affect user-defined template.
